server/controllers/offer/getMyOffers.js

```diff
@@ -1,3 +1,10 @@
+const { getMyOffer } = require('../../database/queries/offers/getMyOffer');
+
 module.exports = (req, res) => {
-  res.send(' get ny offers ');
+  const { memberId } = req.params;
```

Comment:
@alaabadra Here's another place where it would be good to get memberId from the authentication token instead of req.params (or check that the authenticated memberId matches the authenticated member) , once authentication is finished.
Then users can't spy on other users' offers by manually creating requests :)